docs/tables/aws_elasticache_cluster.md

```diff
@@ -85,4 +85,19 @@ select
 from
   aws_elasticache_cluster,
   jsonb_array_elements(security_groups) as sg;
+```
+
+
+### Check Minimum snapshot retention period for Redis cluster is 15 days
```

Comment:
@Priyanka585464 Can we update the example to check `List clusters with automatic backup disabled`?